src/wallet/rpcwallet.cpp

```diff
@@ -2880,6 +2880,9 @@ static RPCHelpMan listunspent()
                             {RPCResult::Type::STR, "scriptPubKey", "the script key"},
                             {RPCResult::Type::STR_AMOUNT, "amount", "the transaction output amount in " + CURRENCY_UNIT},
                             {RPCResult::Type::NUM, "confirmations", "The number of confirmations"},
+                            {RPCResult::Type::NUM, "ancestorcount", /* optional */ true, "The number of in-mempool ancestor transactions, including this one (if transaction is in the mempool)"},
```

Comment:
https://github.com/bitcoin/bitcoin/pull/22903 makes me wonder whether one should change: 

`/* optional */ true` -> `/* optional= */ true`